include/rangers-api/Application/Game/GameModeResourceModule.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+namespace app::game {
+    class GameModeResourceModule : hh::fnd::ReferencedObject {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.